python/py-tmuxp/Portfile

```diff
@@ -48,6 +61,26 @@ if {${name} ne ${subport}} {
              ${worksrcpath}/${subport}
     }
 
+    post-build {
+        system -W ${worksrcpath}/docs "make PYTHONPATH=${worksrcpath}/build/lib SPHINXBUILD=${prefix}/bin/sphinx-build-${python.branch} text man"
+    }
+
+    post-destroot {
+        set doc_build_dir ${worksrcpath}/docs/_build
+
+        xinstall -d ${destroot}${python.prefix}/share/doc
+        xinstall -m 644 {*}[glob ${doc_build_dir}/text/*.txt] \
+            ${destroot}${python.prefix}/share/doc
```

Comment:
typically, the documentation for Python subports goes to: `${destroot}${prefix}/share/doc/${subport}`

Reply:
same applies from man and examples I think; please double check what other ports do

Reply:
Thanks, done